File: java/class 5/Question7.java
import java.util.Scanner;

public class Question7 {
  public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.print("Enter a number: ");
    int number = input.nextInt();
    System.out.print(number % 2 == 0 ? "even": "odd" );
    input.close();
  }
}

File: python/class 5/question 7.py
import re 
num=float(input('Enter a number: '))
num_result = num/2
num_final = str(num_result)
num_final_result = re.search(".5$", num_final)
if num_final_result:
  print('Odd number')
else:
  print('Even number')